nice_updater.sh

```diff
@@ -13,6 +13,8 @@ preferenceFileFullPath="/Library/Preferences/com.github.ryangball.nice_updater.p
 updateInProgressTitle=$(defaults read "$preferenceFileFullPath" UpdateInProgressTitle)
 updateInProgressMessage=$(defaults read "$preferenceFileFullPath" UpdateInProgressMessage)
 loginAfterUpdatesInProgressMessage=$(defaults read "$preferenceFileFullPath" LoginAfterUpdatesInProgressMessage)
+notificationActionButtonText=$(defaults read "$preferenceFileFullPath" notificationActionButtonText)
+notificationOtherButtonText=$(defaults read "$preferenceFileFullPath" notificationOtherButtonText)
```

Comment:
Change the key that is being written to NotificationOtherButtonText (uppercase N) to stay consistent with the other keys in the plist. The variable can keep the lowercase n.